spec/Candid.md

```diff
@@ -864,142 +864,213 @@ service { <methtype>;* } <: service { <methtype'>;* }
 service { <name> : <functype>; <methtype>;* } <: service { <name> : <functype'>; <methtype'>;* }
 ```
 
-### Elaboration
+### Decoding
```

Comment:
The doc speaks of deserialisation in other places. Make consistent?

Reply:
Probably a good time to revisit terminology.
 * Turning the binary form into the abstract value model (probably “decoding”)
 * Turning the textual form into the abstract value model (probably “parsing”)
 * Interpreting an abstract value at an expected type, i.e. this section. But what’s a good term?

Reply:
Sounds good to me. The latter code be "coercing"?